util/src/main/java/org/folio/reservoir/util/readstream/XmlFixerMapper.java

```diff
@@ -18,6 +18,29 @@ public class XmlFixerMapper implements Mapper<Buffer, Buffer> {
 
   int tail;
 
+  int sequenceLength = 0;
+
+  int moved = 0;
+
+  void handleUni(Buffer input, byte leadingByte) {
+    if (leadingByte < -64) {
+      if (sequenceLength > 0) {
+        sequenceLength--;
+      } else {
+        skipByte(input);
+      }
+    } else if (leadingByte < -32) {
```

Comment:
Can we add constants for this that make the code more readable, e.g "1_BYTE_SEQUENCE", etc